### ISSUE TITLE
Can I see your coco learning results?

### ISSUE THREAD

**Issue by km23630** [xuannianz/EfficientDet#72]
I am currently training a model using coco data.
I leave an issue asking if my learning results are going well.

![ap](https://user-images.githubusercontent.com/30399899/74124282-121aec80-4c15-11ea-8c61-c2abd6c4b410.JPG)
![loss](https://user-images.githubusercontent.com/30399899/74124286-134c1980-4c15-11ea-900c-8468d30f0983.JPG)

this is my learning progress.
How much was loss and ap when you learned with coco?
Please share with me.

###
--snapshot imagenet
--phi 1
--gpu 0
--random-transform
--compute-val-loss
--freeze-backbone
--epochs 50
--batch-size 16
--steps 7393
--workers 8
coco
Efficient_det/datasets/coco

this is my training parameters.

**Comment by km23630** [comment]
--snapshot imagenet 
--phi 1 
--gpu 0 
--random-transform 
--compute-val-loss 
--freeze-backbone 
--epochs 50 
--batch-size 16 
--steps 7393 
--workers 8 
coco 
Efficient_det/datasets/coco

this is my training parameters.

**Comment by xuannianz** [comment]
#66
